tag.php

```diff
@@ -0,0 +1,60 @@
+<?php
+	get_header();
+
+	$posts = get_posts( array(
+		'numberposts' => 10,
+		'tag'         => get_queried_object()->slug
+	) );
+
+	if( isset( $posts ) ) {
+		$first_post = array_shift( $posts );
+	}
+
+	if( class_exists( 'UCF_Events_Config' ) ) {
+		$events_feed_url = UCF_Events_Config::get_option_or_default( 'feed_url' );
+		$all_events_link = str_replace( 'feed.json', '', $events_feed_url );
+	}
+?>
+
+<div class="container mt-4 mb-5 pb-sm-4">
+	<div class="row">
+		<div class="col-md-8">
```

Comment:
I think we might want to consider changing this from `col-md-8` to `col-lg-8`. The sidebar content gets a little too narrow in the -md display.
```suggestion
		<div class="col-lg-8">
```